UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -158,6 +158,11 @@ public Acceleration(double meterspersecondsquared)
         /// </summary>
         public static AccelerationUnit BaseUnit => AccelerationUnit.MeterPerSecondSquared;
 
+        /// <summary>
+        ///     The <see cref="BaseDimensions" /> of this quantity.
+        /// </summary>
+        public static BaseDimensions BaseDimensions = new BaseDimensions(1, 0, -2, 0, 0, 0, 0);
```

Comment:
You need to either make it a `readonly` field or add `{ get; } = new ...` to make it a statically initialized getter property. I prefer the latter, mostly because it can be included in an interface or base type later with less changes.

Reply:
Besides this, I think this is getting ready to merge.